src/js/control-bar/volume-control/volume-bar.js

```diff
@@ -112,6 +112,24 @@ class VolumeBar extends Slider {
     this.el_.setAttribute('aria-valuetext', volume + '%');
   }
 
+  /**
+   * When user starts dragging the VolumeBar, store the volume and listen for
+   * the end of the drag. When the drag ends, if the volume was set to zero,
+   * set lastVolume to the stored volume.
+   *
+   * @listens slideractive
+   */
+  updateLastVolume() {
```

Comment:
we probably want this function to be private (only used internally) just like `Player#lastVolume_()`